IPython/core/display.py

```diff
@@ -460,8 +467,9 @@ class DisplayObject(object):
 
     _read_flags = 'r'
     _show_mem_addr = False
+    metadata = None
 
-    def __init__(self, data=None, url=None, filename=None):
+    def __init__(self, data=None, url=None, filename=None, metadata=None):
```

Comment:
Not sure what the expository part of the docstring was originally intended to be, but it's now part of the signature markup.